iset.mm

```diff
@@ -150551,6 +150551,92 @@ Limited Principle of Omniscience (LPO) implies excluded middle, so we
+        2re readdcld rehalfcld eqeltrid cmul 2t2e4 2p2e4 a1i w3a elicc2 sylancl
+        wb simp2d le2addd eqbrtrrid eqbrtrid crp 2rp lemuldivd breqtrrdi simp3d
+        mpbid 4cn 2timesi ledivmuld mpbird elicc2i syl3anbrc ) AEMNOEPQERPQEORU
+        ASZNAECDUBSZOUCSZMKAVQACDAVPMCOMNZRMNZVPMUDUHTORUEUFZHUGZAVPMDWAIUGZUIZ
+        UJUKAOVREPAOOULSZVQPQOVRPQAWERVQPUMAROOUBSVQPUNAOOCDVSAUHUOZWFWBWCACMNZ
+        OCPQZCRPQZACVPNZWGWHWIUPZHAVSVTWJWKUSWFTORCUQURVIZUTADMNZODPQZDRPQZADVP
+        NZWMWNWOUPZIAVSVTWPWQUSWFTORDUQURVIZUTVAVBVCAOVQOWFWDOVDNAVEUOZVFVIKVGA
+        EVRRPKAVRRPQVQORULSZPQAVQRRUBSWTPACDRRWBWCVTATUOZXAAWGWHWIWLVHAWMWNWOWR
+        VHVARVJVKVGAVQROWDXAWSVLVMVCOREUHTVNVO $.
+
+      $( Lemma for ~ sin24dec .  Bounds for ` Y ` .  (Contributed by Jim
+         Kingdon, 26-Feb-2024.) $)
+      sin24declemy01 $p |- ( ph -> Y e. ( 0 (,] 1 ) ) $=
+        ( cc0 c1 wcel cr wbr cle c2 c4 cioc co clt cmin cdiv cicc w3a 2re sylib
+        4re elicc2i simp1d resubcld rehalfcld eqeltrid mpbid halfpos2 breqtrrdi
+        posdifd wb syl a1i simp3d simp2d le2subd 4cn 2cn subaddrii breqtrdi crp
+        2p2e4 2rp divle1le syl2anc mpbird eqbrtrid cxr 1red elioc2 mpbir3and
+        0xr ) AFMNUAUBOZFPOZMFUCQZFNRQZAFDCUDUBZSUEUBZPLAWFADCADPOZSDRQZDTRQZAD
+        STUFUBZOWHWIWJUGISTDUHUJUKUIZULZACPOZSCRQZCTRQZACWKOWNWOWPUGHSTCUHUJUKU
+        IZULZUMZUNUOAMWGFUCAMWFUCQZMWGUCQZACDUCQWTJACDWRWMUSUPAWFPOZWTXAUTWSWFU
+        QVAUPLURAFWGNRLAWGNRQZWFSRQZAWFTSUDUBSRADSTCWMSPOAUHVBTPOAUJVBWRAWHWIWJ
+        WLVCAWNWOWPWQVDVETSSVFVGVGVKVHVIAXBSVJOZXCXDUTWSXEAVLVBWFSVMVNVOVPAMVQO
+        ZNPOWBWCWDWEUGUTXFAWAVBAVRMNFVSVNVT $.
+
+      $( Lemma for ~ sin24dec .  Rearranging the expressions.  (Contributed by
+         Jim Kingdon, 26-Feb-2024.) $)
+      sin24declemsub $p |- ( ph -> ( ( sin ` B ) - ( sin ` A ) )
+          = ( 2 x. ( ( cos ` X ) x. ( sin ` Y ) ) ) ) $=
+        ( csin cfv co caddc cmin c2 cc wcel ccos cmul cdiv cicc wss 2re iccssre
+        c4 cr 4re mp2an sseldi recnd addcld halfcld sincld subcld coscld mulcld
+        eqeltrid pnncand addcomd oveq1d syl5eq a1i oveq12d wa halfaddsub simpld
+        wceq syl2anc eqtrd fveq2d sinadd eqtr3d simprd sinsub 2timesd 3eqtr4d )
+        AEMNZFUANZUBOZEUANZFMNZUBOZPOZWBWEQOZQOWEWEPODMNZCMNZQORWEUBOAWBWEWEAVT
+        WAAEAECDPOZRUCOZSKAWJACDACARUHUDOZUICRUITUHUITWLUIUEUFUJRUHUGUKZHULUMZA
+        DAWLUIDWMIULUMZUNUOUTZUPAFAFDCQOZRUCOZSLAWQADCWOWNUQUOUTZURUSAWCWDAEWPU
+        RAFWSUPUSZWTVAAWHWFWIWGQAEFPOZMNZWHWFAXADMAXADCPOZRUCOZWRPOZDAEXDFWRPAE
+        WKXDKAWJXCRUCACDWNWOVBVCVDZFWRVJALVEZVFADSTZCSTZXEDVJZWOWNXHXIVGZXJXDWR
+        QOZCVJZDCVHZVIVKVLVMAESTZFSTZXBWFVJWPWSEFVNVKVOAEFQOZMNZWIWGAXQCMAXQXLC
+        AEXDFWRQXFXGVFAXHXIXMWOWNXKXJXMXNVPVKVLVMAXOXPXRWGVJWPWSEFVQVKVOVFAWEWT
+        VRVS $.
+
+      $d X t $.
+
+      $( Lemma for ~ sin24dec .  (Contributed by Jim Kingdon, 26-Feb-2024.) $)
+      sin24declemres $p |- ( ph -> ( sin ` B ) < ( sin ` A ) ) $=
```

Comment:
What is the meaning of the suffix "res" of the label here?

Reply:
It stands for "result".

There's often a need to have a lemma which is the same as the overall theorem but has some additional hypotheses which are used for convenience (here, especially `X = ( ( A + B ) / 2 )` and `Y = ( ( B - A ) / 2 )`). I've been naming these with names ending in `lemres` fairly widely but if there is another convention I could switch to that. I'd sort of rather avoid numbered lemmas, not because I see a huge value in making lemma names descriptive, but because it seems to lead to a fair bit of renumbering or `lem2a` style names or the like.

Reply:
According to our conventions, "res" stands for "restriction" (e.g. ~opelres), that's why I was a little confused. 
Maybe sin24declemlt (lt for "less than") would be clearer.

Reply:
`sin24declemlt` it is. Changed.